webrender/src/texture_cache.rs

```diff
@@ -779,6 +779,13 @@ impl TextureCache {
         })
     }
 
+    pub fn add_raw_remove(&mut self, id: TextureId) {
```

Comment:
`add_raw_remove` sounds a bit weird, though it follows the convention. I can't think of an immediately better name now, so I guess this is ok.